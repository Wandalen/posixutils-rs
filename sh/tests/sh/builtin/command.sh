alias a='echo hello'
f () {
  :
}

command -v a
command -v f
command -v set
command -v command

command -V a
command -V f
command -V set
command -V command

command echo test
var=value command :
echo $var
echo () {
  :
}

echo test
command echo test
