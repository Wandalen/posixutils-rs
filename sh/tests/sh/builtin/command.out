alias a='echo hello'
f
set
command
a is aliased to 'echo hello'
f is a function
set is a special shell builtin
command is a shell builtin
test

test
